src/decode/mod.rs

```diff
@@ -34,6 +40,7 @@ impl TraceDecoderKind {
                 #[cfg(decoder_libipt)]
                 return Ok(());
             }
+            Self::YkPT => return Ok(()),
```

Comment:
`return` isn't needed here.

Reply:
Fixed in [3567abf](https://github.com/ykjit/hwtracer/pull/105/commits/3567abfb722b6106c105a07bc9a0b4ceda47d3a4)